packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -323,21 +320,18 @@ export const prepareStatusManager = (
       },
 
       /**
-       * Mark a transaction as `FORWARDED`
+       * Mark a transaction as `FORWARDED` or `FORWARD_FAILED`
        *
-       * @param {EvmHash | undefined} txHash - undefined in case mint before observed
-       * @param {NobleAddress} nfa
-       * @param {bigint} amount
+       * @param {EvmHash} txHash
+       * @param {boolean} success
        */
-      forwarded(txHash, nfa, amount) {
-        if (txHash) {
-          void publishTxnRecord(txHash, harden({ status: TxStatus.Forwarded }));
-        } else {
-          // TODO store (early) `Minted` transactions to check against incoming evidence
-          log(
-            `⚠️ Forwarded minted amount ${amount} from account ${nfa} before it was observed.`,
-          );
-        }
+      forwarded(txHash, success) {
```

Comment:
boolean params are a smell IME.

I think this function isn't worth its weight anymore. just inline the `publishTxnRecord` call

Reply:
> just inline the publishTxnRecord call

I don't follow this suggestion - can you rephrase?

Reply:
remove this `forwarded` function and in the places it's called do what it was doing.

Reply:
Ok, but the Settler does not have access to `StatusManager`'s vstorage writer currently. Are you suggesting we export `publishTxnRecord`?

Reply:
I missed that this was an external method. my mistake! resolving